frontend/src/routes/Login.tsx

```diff
@@ -314,6 +314,7 @@ const Field: React.FC<FieldProps> = ({ isEmpty, children }) => {
                 padding: "16px 16px",
                 border: `1px solid ${COLORS.grey4}`,
                 borderRadius: 4,
+                ":autofill": { background: isDark ? "#2f3644" : "#e7f0fe" },
```

Comment:
Why are we using fixed colors here? Will they still work fine when any of the configurable colors are changed?

Reply:
These are the background colors that chrome uses for autofill. Firefox uses a rather ugly yellow (though it's still alright compared to Safari).
Firefox yellow:
<img width="452" alt="Bildschirmfoto 2023-06-21 um 11 06 46" src="https://github.com/elan-ev/tobira/assets/94838646/afd0b9f8-df8f-447a-8b97-3ba3016457d6">
Chrome blue (in Firefox):
<img width="452" alt="Bildschirmfoto 2023-06-21 um 11 07 02" src="https://github.com/elan-ev/tobira/assets/94838646/42019ed6-7e70-4585-b7e5-79099328f2a0">

Unfortunately, Chrome's user agent stylesheet uses `!important` all over the place, so I can't override their text color for `autofill` fields (which is too bright).

Side note: check out Safari's autofill background color.... it's soo bad :/ and I have no idea how to change that:
<img width="452" alt="Bildschirmfoto 2023-06-21 um 11 31 37" src="https://github.com/elan-ev/tobira/assets/94838646/c0a135b2-c7d9-4d2c-ad1c-5586b946a36b">

Reply:
Mh ... so browsers use some built-in colors for auto filled fields. And we want to override it, I assume? But that still leaves the question: why are you using these specific colors? Why not some grey or primary color?

Reply:
I used these because I think chrome's colors are rather fitting.
`primary0` and doesn't work well in my opinion, looks weird in an input field. I guess we could use some grey if you would prefer that?

Reply:
I mean we don't __have__ to override this, but I kinda sticks out, doesn't it? I assume most users will use autofill... Though since this only applies for Firefox anyway we could ignore it.

Reply:
I don't mind these colors specifically, the big question is simply: do they work well with any other colors that could be configured? Like what if primary is green, orange, pink, ...? Have you thought about that? Would you say it shouldn't be a problem?
If we keep those, a comment might be appropriate to explain it. I think this is the only place where we use specific colors that are not derived from the configured ones (ignoring pure black and white).